src/BUILD.bazel

```diff
@@ -35,32 +36,36 @@ ts_devserver(
     # serve these files rooted at /
     additional_root_paths = [
         "npm/node_modules/zone.js/dist",
-        "npm/node_modules/tslib",
         "npm/node_modules/@angular/material/prebuilt-themes",
         "npm/node_modules/@ngrx/store/bundles",
     ],
     # Start from the development version of the main
     entry_module = "angular_bazel_example/src/main.dev",
+    index_html = "index.html",
+    # These scripts will be included in the JS bundle after require.js
+    # They should have only named UMD modules, or require.js will throw.
```

Comment:
nit: Also add that they can also be plain old javascript like `require.config.js` and `module-id.js`